### DISCUSSION TITLE
Fcitx5 on weston fsl-imx-xwayland

### DISCUSSION THREAD

**Discussion by LuThaiHoc** [fcitx/fcitx5#719] [Q&A]
Hi. Im trying to use fcitx5 on weston wayland.
When building image via Yocto, I follow this link to build fcitx5 input method and its engines: https://github.com/ashie/meta-inputmethod
The built was successed, but I cannot type any specical language.
My configuration: 
**weston.ini:** 
```
[input-method]
path=/usr/bin/fcitx5
```
**/etc/environment**
```
export XMODIFIERS=@im=fcitx
export GTK_IM_MODULE=fcitx
export QT_IM_MODULE=fcitx
```
**~/.config/fcitx5/profile** (auto generated)
```
[Groups/0]
# Group Name
Name=Default
# Layout
Default Layout=us
# Default Input Method
DefaultIM=anthy

[Groups/0/Items/0]
# Name
Name=keyboard-us
# Layout
Layout=

[Groups/0/Items/1]
# Name
Name=anthy
# Layout
Layout=

[Groups/0/Items/2]
# Name
Name=chewing
# Layout
Layout=

[Groups/0/Items/3]
# Name
Name=hangul
# Layout
Layout=

[GroupOrder]
0=Default
```
Result of fcitx5-diagnose: 
[diagnose.txt](https://github.com/fcitx/fcitx5/files/10728446/diagnose.txt)

I see that fcitx5-remote always result 0 although I tried to active input method via -o options
Can someone help me to debug this? Thank you!

**Comment by Vamp898** [comment]
Hi there,

i am trying to help you with that, my issue is that even when i start Weston with the dbus-launch command, fcitx5-configtool complains, that DBus is not running (it is). Not sure what exactly i am doing wrong but something is up with DBus.

I can see the Daemon running in the background (DBus too) but i can not type anywhere nor do any configuration.

Maybe that is the same issue you have too? Not sure. Can you verify the output of the fcitx5-configtool ?
